rnd/autogpt_server/autogpt_server/server/routers/analytics.py

```diff
@@ -0,0 +1,138 @@
+"""Analytics API"""
+
+from typing import Annotated, Optional
+
+import fastapi
+import prisma
+import prisma.enums
+import pydantic
+
+import autogpt_server.data.analytics
+from autogpt_server.server.utils import get_user_id
+
+router = fastapi.APIRouter()
+
+
+class UserData(pydantic.BaseModel):
+    user_id: str
+    email: str
+    name: str
+    username: str
+
+
+@router.post(path="/log_new_user")
+async def log_create_user(
```

Comment:
Why do we need an endpoint for this, when we can determine this from the data we already have

Reply:
Totally agree. This is designed to include other data as well such as referrals when implemented